documentation/contribute/index.qmd

```diff
@@ -2,4 +2,22 @@
 title: Contribute
 description: Contribute a dataset, method or metric to OpenProblems
 order: 20
----
\ No newline at end of file
+---
+
+To contribute new functionality to OpenProblems, please follow the steps below:
+
+1. Search through the [GitHub Issues](https://github.com/openproblems-bio/openproblems-v2/issues)  tracker to ensure that no one else is already working on the feature you would like to add. If someone is already working on it, post in that issue to express your interest or reach out to the contributors working on the issue directly.
+
+2. If there is no existing issue tracking the feature, create a new one.
+
+3. [Install the necessary requirements](requirements.qmd) to work with OpenProblems.
+
+4. Follow the [getting started](first_time_setup.qmd) guide to fork and clone the repository and download the necessary test resources.
```

Comment:
```suggestion
4. Follow the [getting started](getting_started.qmd) guide to fork and clone the repository and download the necessary test resources.
```

Reply:
Oh, I didn't know you could make suggestions like this!

Reply:
I accidentally fixed it manually before realising I could click a button to do it for me.